templates/apollo-starter-ts/generated/resolvers.ts

```diff
@@ -0,0 +1,11 @@
+export const resolvers = {
```

Comment:
Add git ignore for generated folder outside template - in root of the repo!

Reply:
Or do we want this to be there so it will work without generation?

Reply:
IF yes then we need to documenent that.

Reply:
This was for mapping. Once we do `generate` it will be replaced.